src/lib.rs

```diff
@@ -280,10 +331,14 @@ struct Context<'a> {
     face: Face<'a>,
     /// A map from old gids to new gids, and the reverse
     mapper: GlyphRemapper,
-    /// The kind of face.
-    kind: FontKind,
+    /// The font flavor.
+    flavor: FontFlavor,
     /// Subsetted tables.
     tables: Vec<(Tag, Cow<'a, [u8]>)>,
+    pub(crate) interjector: Interjector<'a>,
```

Comment:
I think of all the fields the interjector is the one that most deserves a comment :p

Reply:
Also, I don't think there is a need for pub(crate) here, as we are already in lib.rs.